apis/FrameRate/IFrameRate.h

```diff
@@ -0,0 +1,115 @@
+
+// @stubgen:include <com/IIteratorType.h>
+
+namespace WPEFramework
+{
+    namespace Exchange
+    {
+        /* @json 1.0.0 @text:keep */
+        struct EXTERNAL IFrameRate : virtual public Core::IUnknown
+        {
+            enum { ID = ID_FRAMERATE };
+
+            // @event
+            struct EXTERNAL INotification : virtual public Core::IUnknown
+            {
+                enum { ID = ID_FRAMERATE_NOTIFICATION };
+
+                // @text onFpsEvent
+                // @brief Triggered by callback from FrameRate after onFpsEvent
+                // @param average - in - int
+                // @param min - in - int
+                // @param max - in - int
+                virtual void OnFpsEvent(int average, int min, int max) {};
+
+                // @text onDisplayFrameRateChanging
+                // @brief Triggered when the framerate changes started
+                // @param displayFrameRate - in - string
+                virtual void OnDisplayFrameRateChanging(const string& displayFrameRate) {};
+
+                // @text onDisplayFrameRateChanged
+                // @brief Triggered when the framerate changed.
+                // @param displayFrameRate - in - string
+                virtual void OnDisplayFrameRateChanged(const string& displayFrameRate) {};
+            };
+
+            virtual Core::hresult Register(IFrameRate::INotification* notification /* @in */) = 0;
+            virtual Core::hresult Unregister(IFrameRate::INotification* notification /* @in */) = 0;
+
+            /** Gets the Display Frame Rate*/
+            // @text getDisplayFrameRate
+            // @brief Gets the current display frame rate values.
+            // @param framerate - out - string
+            // @param success - out - boolean
+            virtual Core::hresult GetDisplayFrameRate(string& framerate /* @out */, bool& success /* @out */) = 0;
+
+            /** Gets framerate mode */
+            // @text getFrmMode
+            // @brief Gets the current auto framerate mode.
+            // @param frmmode - out - int
+            // @param success - out - boolean
+            virtual Core::hresult GetFrmMode(int &framerateMode /* @out @text:auto-frm-mode */, bool& success /* @out */) = 0;
+
+            /** Sets the FPS data collection interval */
+            // @text setCollectionFrequency
+            // @brief Sets the FPS data collection interval.
+            // @param frequency - in -  int
+            // @param success - out - boolean
+            virtual Core::hresult SetCollectionFrequency(int frequency /* @in */, bool& success /* @out */) = 0;
+
+            /** Sets the display framerate values */
+            // @text setDisplayFrameRate
+            // @brief Sets the display framerate values.
+            // @param framerate - in - string
+            // @param success - out - boolean
+            virtual Core::hresult SetDisplayFrameRate(const string& framerate /* @in */, bool& success /* @out */) = 0;
```

Comment:
Do not tag "in" param.
Does this result in returning the following result object?
"result": {
"success": true
} or returning "result: true"? If it returns later ,would it affect the caller (because the existing code is returning former format).